feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/radio.c

```diff
@@ -847,6 +872,7 @@ apc_cld_mon_cb(struct schema_Manager *mgr)
 		LOGI("The return value: %d\n", WEXITSTATUS(ret));
```

Comment:
I see this log a lot on my AP.  It needs to be more informative and lower priority (like DEBUG or TRACE).

Reply:
done